.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+jdk:
+- openjdk8
+- oraclejdk8
+- oraclejdk9
+script: ./gradlew build compileMinJosmVersion && cp -T `find build/libs/*.jar | head -1` "build/libs/fieldpapers.jar"
+deploy:
+  provider: releases
+  api_key:
+    secure: PutYourEncryptedApiKeyHere
```

Comment:
@iandees To enable GitHub releases, you'll just need to fill in your encrypted API key here.

Reply:
@floscher do you think I can reuse the encrypted API key from the other repo or is does it need to be encryped once per repo?

Reply:
I think you can reuse the API key if you want (and still know it 😉) but have to encrypt it again. AFAIR the keys to encrypt/decrypt vary from repo to repo, so that e.g. I can't upload builds to your Travis account from my fork.